src/uwtools/cli.py

```diff
@@ -160,7 +160,7 @@ def _add_subparser_config_validate(subparsers: Subparsers) -> SubmodeChecks:
     required = parser.add_argument_group(TITLE_REQ_ARG)
     _add_arg_schema_file(required)
     optional = _basic_setup(parser)
-    _add_arg_input_file(optional)
+    _add_arg_config_file(optional)
```

Comment:
So that was my first edit version - changing to `STR.infile` is consistent with the other validate entry, but I was getting some squirrely issues on the `test__dispatch_config_validate_config_obj` in test_cli

Reply:
If you make and push that change, I'd be happy to fetch it and help diagnose.

Reply:
Push is up - the API should be fine, I think the patching is just off